release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,890 @@
+specification complaint container registry for hosting OpenShift content.
+Administrators first need replicate content from Quay.io to their local
+container registry. Once that is done, `openshift-install` can be configured to
+generate ignition configurations that pull content locally rather than from
+Quay.io. This is designed to work with user-provisioned infrastructure
+deployments only.
+
+For disconnected environments, you are still able to use Operators from the OLM
+catalog. However, you must manually pull the Operator sources in order to
+populate the the offline catalog. This manual process is only a temporary
+workaround for {product-title}. 4.2. An automated solution is coming in a future
+release.
+
+[id="ocp-4-2-cluster-wide-egresss-proxy"]
+==== Cluster-wide egress proxy
+
+{product-title} 4.2 introduces support for installing and updating an OpenShift
+cluster through a corporate proxy server. Proxy information (httpProxy,
+httpsProxy, and noProxy) can be defined in `install-config`, which is used
+during the installation process and can also be managed post-installation via
+the cluster proxy object.
+
+Also, there is now support for providing your own CA bundles allowing the
+corporate proxy to MITM HTTPS.
+
+[id="ocp-4-2-new-platform-boundary"]
+==== New platform boundary
+
+{product-title} is now aware of the entire infrastructure and brings the
+operating system under management. This makes installation and updates seamless
+across the platform and the underlying operating system. Everything is managed
+as a single entity.
+
+[id="ocp-4-2-IPI-UPI"]
+==== IPI and UPI
+
+In {product-title} {product-version}, there are two primary installation
+experiences: Full stack automation (IPI) and pre-existing infrastructure (UPI).
+
+With full stack automation, the installer controls all areas of the installation
+including infrastructure provisioning with an opinionated best practices
+deployment of {product-title}. With re-existing infrastructure deployments,
+administrators are responsible for creating and managing their own
+infrastructure allowing greater customization and operational flexibility.
+
+[id="ocp-4-2-full-stack-automated-deployments"]
+==== Full stack automated deployments
+
+In {product-title} {product-version}, there is expanded support for full stack
+automated deployments to include AWS, Microsoft Azure, Google Cloud Platform (GCP),
+and Red Hat OpenStack Platform, as well as adding GCP to the existing list of
+user provisioned infrastructure supported providers that already includes AWS,
+Bare Metal, and VMware vSphere.
+
+[id="ocp-4-2-operators"]
+=== Operators
+
+[id="ocp-4-2-operator-installation"]
+=== Operator installation
+
+Previously, only users carrying `cluster-admin` roles  were allowed to install
+Operators. In {product-title} 4.2, the `cluster-admin` can select namespaces in
+which namespace administrators can install Operators self-sufficiently. The
```

Comment:
Now, the namespace administrators cannot see the `OperatorHub` on the Web console. That means, they cannot install the Operators by themselves on the Web console actually. cc: @spadgett

Reply:
Right. Project admins can see installed operators and create resources for those operators, but they can't install new operators from OperatorHub.